python/pyspark/pandas/groupby.py

```diff
@@ -3762,7 +3762,7 @@ def _apply_series_op(
             return psser.copy()
 
     def _cleanup_and_return(self, psdf: DataFrame) -> Series:
```

Comment:
Should probably rename this to something else ... e.g., `_handle_output`

Reply:
nice, let me update it